internal/log_analysis/log_processor/parsers/awslogs/waf_web_acl.go

```diff
@@ -0,0 +1,100 @@
+	RateBasedRuleList           []RateBasedRuleListDetail `json:"rateBasedRuleList" description:"The list of rate-based rules that acted on the request."`
+	RuleGroupList               []RuleGroupListDetail     `json:"ruleGroupList" description:"The list of rule groups that acted on this request. In the preceding code example, there is only one."`
+	TerminatingRuleID           pantherlog.String         `json:"terminatingRuleId"`
+	TerminatingRuleMatchDetails []RuleMatchDetail         `json:"terminatingRuleMatchDetails" description:"Detailed information about the terminating rule that matched the request. A terminating rule has an action that ends the inspection process against a web request. Possible actions for a terminating rule are ALLOW and BLOCK. This is only populated for SQL injection and cross-site scripting (XSS) match rule statements. As with all rule statements that inspect for more than one thing, AWS WAF applies the action on the first match and stops inspecting the web request. A web request with a terminating action could contain other threats, in addition to the one reported in the log."`
+	TerminatingRuleType         pantherlog.String         `json:"terminatingRuleType" description:"The type of rule that terminated the request. Possible values: RATE_BASED, REGULAR, GROUP, and MANAGED_RULE_GROUP."`
+	Timestamp                   pantherlog.Time           `json:"timestamp" tcodec:"unix_ms" event_time:"true" description:"The timestamp in milliseconds."`
+	WebACLID                    pantherlog.String         `json:"webaclId" description:"The GUID of the web ACL."`
+}
+
+// nolint:lll,maligned
+type RuleGroupListDetail struct {
+	ExcludedRules               []ExcludedRule    `json:"excludedRules" description:"The list of rules in the rule group that you have excluded. The action for these rules is set to COUNT."`
+	NonTerminatingMatchingRules []RuleDetail      `json:"nonTerminatingMatchingRules"`
+	RuleGroupID                 pantherlog.String `json:"ruleGroupId"`
+	TerminatingRule             *RuleDetail       `json:"terminatingRule"`
+}
+
+// nolint:lll,maligned
+type ExcludedRule struct {
+	ExclusionType pantherlog.String `json:"exclusionType" description:"A type that indicates that the excluded rule has the action COUNT (most likely value is EXCLUDED_AS_COUNT)."`
+	RuleID        pantherlog.String `json:"ruleId" description:"The ID of the rule within the rule group that is excluded."`
+}
+
+// nolint:lll,maligned
+type RuleDetail struct {
+	RuleID           pantherlog.String  `json:"ruleId" description:"The Rule ID."`
+	Action           pantherlog.String  `json:"action" description:"The configured rule action. For non-terminating rules the value is always COUNT."`
+	RuleMatchDetails *[]RuleMatchDetail `json:"ruleMatchDetails" description:"Detailed information about the rule that matched the request. This field is only populated for SQL injection and cross-site scripting (XSS) match rule statements."`
```

Comment:
Slices do not need to be pointers to be omitempty
```suggestion
	RuleMatchDetails []RuleMatchDetail `json:"ruleMatchDetails" description:"Detailed information about the rule that matched the request. This field is only populated for SQL injection and cross-site scripting (XSS) match rule statements."`
```

Reply:
You are right, tests pass.

Reply:
Fixed in https://github.com/panther-labs/panther/pull/2403/commits/6cb625ed946f878b556621ddcefb65bc65af091d